### ISSUE TITLE
Indicator issues

### ISSUE THREAD

**Issue by XiaoHaoPan** [CVMI-Lab/VideoDemoireing#7]
The metrics I used to test the 59th epoch with [iPhone](https://www.dropbox.com/sh/jalr5860ukesgvl/AAC2ozt00TgzWzSgFTRG-z8ma?dl=0) training were: Avg. LPIPS is 0.2057, PSNR is 21.9625 and SSIM is 0.7185

The metrics I used to test the 59th epoch with [TCL](https://www.dropbox.com/sh/9yqd5ui9h7cjwy3/AADSn8AFEP4ewzk3QFdwpz8-a?dl=0) training were:Avg. LPIPS is 0.2025, PSNR is 21.0502 and SSIM is 0.7367

All seem to differ from the metrics in the paper.I use the same data_v1.

**Comment by XiaoHaoPan** [comment]
In addition, I noticed that the learning rate of the pre-trained model of TCL and IPhone provided by you is not the same at the 50th epoch, and the validation metrics seem to be quite high. Do you have different learning rate scheduling for data taken by two different devices?

![image](https://github.com/CVMI-Lab/VideoDemoireing/assets/127167436/fbd6c72a-eacc-4896-8786-51d496737203)

**Comment by 13055068915** [comment]
> In addition, I noticed that the learning rate of the pre-trained model of TCL and IPhone provided by you is not the same at the 50th epoch, and the validation metrics seem to be quite high. Do you have different learning rate scheduling for data taken by two different devices?
> 
> ![image](https://github.com/CVMI-Lab/VideoDemoireing/assets/127167436/fbd6c72a-eacc-4896-8786-51d496737203)

师兄您好，我看到您24年在复现VideoMorieing项目，然后我想请问您他这个项目限定用很低的cuda版本，对应很低的gpu，我的服务器cuda版本太高跑不了，然后我尝试将DCNv2用其他函数替换，目前还在修改，请问您是怎么做的，我看您应该是成功test了

**Comment by XiaoHaoPan** [comment]
> > 此外，我注意到你提供的 TCL 和 iPhone 的预训练模型的学习率在第 50 个 epoch 并不相同，验证指标似乎相当高。对于两个不同设备获取的数据，您是否有不同的学习率计划？
> > ![image](https://github.com/CVMI-Lab/VideoDemoireing/assets/127167436/fbd6c72a-eacc-4896-8786-51d496737203)
> 
> 师兄您好，我看到您24年在复现VideoMorieing项目，然后我想请问您他这个项目限定用很低的cuda版本，对应很低的gpu，我的服务器cuda版本太高跑不了，然后我尝试将DCNv2用其他函数替换，目前还在修改，请问您是怎么做的，我看您应该是成功测试了

你可以重新编译DCNv2，从[这里](https://github.com/lucasjinreal/DCNv2_latest)
